web/src/app/administration/workbasket/details/information/workbasket-information.component.spec.ts

```diff
@@ -138,7 +138,7 @@ describe('WorkbasketInformationComponent', () => {
   }));
 
   it('should post a new workbasket when no workbasketId is defined and update workbasket', async(() => {
-    const workbasket = new Workbasket(undefined, 'created', 'keyModified', 'domain', ICONTYPES.TOPIC, 'modified', 'name', 'description',
+    const workbasket = new Workbasket('', 'created', 'keyModified', 'domain', ICONTYPES.TOPIC, 'modified', 'name', 'description',
```

Comment:
is undefined the same as an empty string? what about null?

Reply:
It's not the same but undefined is better. So I took this changes back